services/QuillLMS/engines/evidence/lib/evidence/evidence/change_log.rb

```diff
@@ -63,12 +63,25 @@ def activity_versions
       Evidence.change_log_class.where(changed_record_id: id, changed_attribute: 'version', changed_record_type: 'Evidence::Activity').map do |row|
         {
           note: row.explanation,
-          updated_at: row.updated_at,
-          new_value: row.new_value
+          created_at: row.created_at,
+          new_value: row.new_value,
+          sessions: version_sessions(row, id).count
```

Comment:
For better transparency, I feel like this key should be called `session_count`.  I would also rename `version_sessions` to `session_count` and have that method just return the count itself rather than the relation.